src/objects/Trigger_Handler__c.object

```diff
@@ -89,6 +89,16 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Exclude_For_These_Usernames__c</fullName>
```

Comment:
Hi @smarovich. Did you happen to see my note from a couple of days ago on approving the inline help/description for this new field? Here is a direct link to the field definition in the PR: https://github.com/SalesforceFoundation/Cumulus/pull/2584/files#diff-e5e3591142b5df80da074261b1a9114d